.github/workflows/sdks.yml

```diff
@@ -480,3 +484,85 @@ jobs:
           api-level: 29
           arch: x86_64
           script: ~/test-toolchain.sh
+      - name: Get cached SDK (aarch64)
+        if: ${{ matrix.os == 'ubuntu-22.04' && matrix.arch == 'x86_64' }}
+        uses: actions/cache/restore@v4
+        with:
+          path: ~/swift-${{ matrix.version }}-android-aarch64-*-sdk.tar.xz
+          key: ${{ steps.version.outputs.tag }}-aarch64-ndk-27b-sdk
+          fail-on-cache-miss: true
+      - name: Get cached SDK (armv7)
+        if: ${{ matrix.os == 'ubuntu-22.04' && matrix.arch == 'x86_64' }}
+        uses: actions/cache/restore@v4
+        with:
+          path: ~/swift-${{ matrix.version }}-android-armv7-*-sdk.tar.xz
+          key: ${{ steps.version.outputs.tag }}-armv7-ndk-27b-sdk
+          fail-on-cache-miss: true
+      - name: Assemble SDK
+        if: ${{ matrix.os == 'ubuntu-22.04' && matrix.arch == 'x86_64' }}
+        id: assemble-sdk
+        run: |
+          set -x
+          ls -lah ~/swift-*-sdk.tar.xz
+
+          # e.g. swift-5.10.1-RELEASE-android-sdk
+          SDK_DIR=${{ steps.version.outputs.tag }}-android-sdk
+          echo "Merging $(echo $(ls -1)) into SDK_DIR: ${SDK_DIR}"
+
+          mkdir ${SDK_DIR}
+
+          for SDK_PATH in ~/swift-*-sdk.tar.xz; do
+            MERGEDIR=$(basename $SDK_PATH .tar.xz)
+            mkdir ${MERGEDIR}
+            pushd ${MERGEDIR}
+            # e.g. swift-android-sdk-release-aarch64/swift-release-android-aarch64-24-sdk.tar.xz
+            tar xf ${SDK_PATH}
+            tree swift-*-sdk
+            pushd swift-*-sdk
+
+            ARCH=$(ls usr/lib/swift_static/android/*/swiftrt.o | cut -f 5 -d '/')
+
+            TRIPLE="${ARCH}-linux-android"
+            if [[ $ARCH == 'armv7' ]]; then
+                TRIPLE="arm-linux-androideabi"
+            fi
+
+            echo "Creating ${TRIPLE} from ${MERGEDIR}…"
+
+            # remove the destination files, which include invalid absolute paths
+            rm usr/swiftpm-*.json
+
+            mkdir -p usr/lib/${TRIPLE}
+            rm -r usr/bin
+            mv usr/lib/swift_static/android/*.a usr/lib/swift_static/android/${ARCH}/
```

Comment:
Add a line after this:
```
patchelf --set-rpath \$ORIGIN usr/lib/swift/android/lib[FXs]*.so
```
This will make sure the Swift SDK libraries have the right runpath after this pull.